src/components/common/HeadersComponent/HeadersComponent.js

```diff
@@ -35,7 +35,7 @@ const HeadersComponent = (props) => {
   return (
     <HeaderReaderWrap>
       <ReadingIconStyleRepresentation>
-        <IconRepresentation src={image} alt={alt} />
+        {image && <IconRepresentation src={image} alt={alt} />}
```

Comment:
as image is a string (not boolean or null if no image) please update condition:
```suggestion
        {image ? <IconRepresentation src={image} alt={alt} /> : null}
```